src/lang/Messages.properties

```diff
@@ -71,6 +71,7 @@ api.dialogue.browseApiNotEnabled.title = API Not Enabled
 api.dialogue.browseApiNotEnabled.message = The API must be enabled before you can browse it.\nDo you want to enable the API?
 api.dialogue.browseApiNotEnabled.button.confirm.label = Enable and Browse the API
 api.dialogue.browseApiNotEnabled.button.cancel.label = Cancel
+api.deprecated.option.endpoint = Option no longer in effective use.
```

Comment:
I couldn't immediately think of a good alternative as I read this, I'll keep it in the back of my head and if I think of some good text add it here. If you have another idea when that's great too 👍